.github/workflows/build.yml

```diff
@@ -14,14 +17,17 @@ jobs:
     runs-on: ubuntu-latest
     permissions:
       contents: write
+    strategy:
```

Comment:
You're using a matrix but then only setting one version, so it's adding complexity that's not used. Why not simply keep the `node-version: 16` as before?